src/codegeneration/SymbolTransformer.js

```diff
@@ -96,6 +100,25 @@ function isSafeTypeofString(tree) {
  */
 export class SymbolTransformer extends TempVarTransformer {
 
+  prependRuntimeOption_(scriptItemList) {
+    return [
```

Comment:
This looks like the cause of the test failure. Use prependStatement.js. It handles strict mode.